lib/components/FullBanner/FullBanner.tsx

```diff
@@ -0,0 +1,67 @@
+import { PageHeader } from '../PageHeader/PageHeader'
+import { justifyContentClasses } from '../../utils/propClasses'
+
+type justifyContentKeys = keyof typeof justifyContentClasses
+
+const opacityValues = Array.from({ length: 21 }, (_, index) => 60 + index)
+
+export interface FullBannerProps {
+  children?: React.ReactNode
+  as?: 'section' | 'div'
+  title: string
+  image?: string
+  focalPointX?: string
+  focalPointY?: string
+  justify?: justifyContentKeys
+  opacity?: (typeof opacityValues)[number]
+}
+
+// Used in WideImage, make a helper function when moving this component into RDS
+const getBackgroundImageStyles = (image: string = '', focalPointX: string, focalPointY: string) => ({
+  backgroundImage: `url(${image})`,
+  backgroundPosition: `${focalPointX}% ${focalPointY}%`,
+})
+
+export const FullBanner = ({
+  children,
+  as = 'section',
+  title,
+  image,
+  focalPointX = '50',
+  focalPointY = '50',
+  opacity = 70,
+  justify = 'start',
+}: FullBannerProps) => {
+  if (!opacityValues.includes(opacity)) {
+    console.warn(`Invalid opacity value: ${opacity}. It should be one of ${opacityValues.join(', ')}.`)
+  }
+
+  const FullBannerComponent = as
+  const inlineStyle = getBackgroundImageStyles(image, focalPointX, focalPointY)
+  const opacityStyle = {
+    opacity: `0.${opacity}`,
+  }
+  const contentAlign = justify === 'center' ? 'text-center [&>*]:justify-center' : ''
+
+  return (
+    <FullBannerComponent
+      className={`cu-fullbanner cu-fullbanner--${justify} cu-browsers-edge not-contained not-prose overflow-hidden bg-cu-black-200 bg-cover mb-6 -mt-4 md:-mt-6 lg:-mt-12 pt-80 pb-0 md:pt-48 md:pb-12 lg:pt-64 lg:pb-24 xl:pt-96 xl:pb-36 px-0 md:px-10`}
```

Comment:
instead of the margin classes, can we set the padding classes to wrapper of inner children?

Reply:
This was replaced in another branch with css to target first-child so the banner removes spacing only when first inside main or section